memcache/memcache.go

```diff
@@ -132,6 +132,9 @@ func NewFromSelector(ss ServerSelector) *Client {
 // Client is a memcache client.
 // It is safe for unlocked use by multiple concurrent goroutines.
 type Client struct {
+	// Dialer specifies a custom dialer used to dial new connections to a server.
```

Comment:
`// DialTimeout`, not `// Dialer`.

But the more idiomatic thing to do these days would be to use `DialContext func(ctx context.Context, network, address string) (Conn, error)` and have us pass in a context instead.